server/src/main/java/org/opensearch/index/IndexSettings.java

```diff
@@ -55,6 +55,7 @@
 import org.opensearch.ingest.IngestService;
 import org.opensearch.node.Node;
 import org.opensearch.node.remotestore.RemoteStoreNodeAttribute;
+import org.opensearch.search.SearchService;
```

Comment:
nit: Can you import just the static values instead of the entire `SearchService` class?

Reply:
good point